tests/test_aws_region_name_setting.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from powerlibs.aws.process_status_notifier import ProcessStatusNotifier
+
+
+def test_aws_region_name_setting():
+    aws_region = os.environ.get('AWS_REGION_NAME') or os.environ['AWS_REGION']
```

Comment:
I don't think it's interesting to test whether the right environment variable was used, specially when the method to do so is basically the same as in the code.
Besides, it's a dubious test, since only one condition is going to be tested, and not the another, and that depends on the system the test is run.

I'd rather mock `os.environ` and write  three different tests: `with_aws_region_name, `with_aws_region` and `without_aws_region_at_all`.